articles/integrations/aws/index.md

```diff
@@ -21,7 +21,7 @@ useCase:
 
 <ul class="topic-links">
   <li>
-    <i class="icon icon-budicon-715"></i><a href="/integrations/aws/sso">Enable SSO to the AWS Console</a>
+    <i class="icon icon-budicon-715"></i><a href="/integrations/aws/sso">Configure SSO to the AWS Console</a>
```

Comment:
This is not consistent with other language that says "Configure SSO with the AWS Console.